nni/retiarii/nn/pytorch/mutator.py

```diff
@@ -47,7 +47,12 @@ def mutate(self, model):
         n_candidates = self.nodes[0].operation.parameters['n_candidates']
         n_chosen = self.nodes[0].operation.parameters['n_chosen']
         candidates = list(range(n_candidates))
-        chosen = [self.choice(candidates) for _ in range(n_chosen)]
+        if n_chosen is None:
+            chosen = [i for i in candidates if self.choice([False, True])]
+            # This is a hack to make choice align with the previous format
+            self._cur_samples = chosen
```

Comment:
what is `self._cur_samples` used for? it is assigned only in `if` not `else`

Reply:
It's explained in L52. For example, it will convert `[False, True, True]` into `[1, 2]`.

Reply:
maybe, add "FIXME" is better, because it is too hacky